plutus-core/index-envs/src/Data/DeBruijnEnv.hs

```diff
@@ -38,9 +38,9 @@ instance DeBruijnEnv (BRAL.RAList a) where
     {-# INLINABLE cons #-}
     cons = BRAL.Cons
     {-# INLINABLE index #-}
-    index = BRAL.safeIndexOne
+    index e i = BRAL.safeIndexZero e (i-1)
```

Comment:
I meant do the subtraction in the CEK machine. The point is that `index` methods usually are indexed from 0. So either we need to document in the class method doc that it's 1-based (at which point you might as well use the specialized version if it's faster), or have it be 0-based and do the subtraction in the machine.